### ISSUE TITLE
Consider tidyverse piping interface

### ISSUE THREAD

**Issue by markfairbanks** [markfairbanks/tidypolars#208]
Proposed interface:

```python
import polars as pl
from tidypolars import col, arrange, filter, mutate

df = pl.DataFrame(dict(x = range(3), y = range(3), z = range(3)))

(
    df >>
    arrange('x') >>
    filter(col('x') <= 2) >>
    mutate(double_x = col('x') * 2)
)
```

Positives:
* Would work directly on polars DataFrame
  * Wouldn't need a special Tibble class
  * Would allow more easily mixing polars/tidypolars syntax
* Would work on polars LazyFrame
* Issues like #204 wouldn't occur

Negatives:
* Not very pythonic

**Comment by markfairbanks** [comment]
Another option: https://github.com/pola-rs/polars/pull/5531

This would allow using a `tp` namespace on a polars data frame

``` python
df = pl.DataFrame({"a": [1, 2], "b": [3, 4]})

(
    df
    .tp.mutate(double_a = col('a') * 2)
    .tp.filter(col('a') < 3)
    .tp.arrange('a', 'b')
)
```

**Comment by alexander-beedie** [comment]
I think a `tidy` namespace would be ideal here, now that you can register them - it's ideal for library authors (you get to extend the relevant classes _without_ subclassing/mixins _or_ requiring changes in the core).

Also, the `<<` and `>>` operators may get used for bitshift ops (their usual meaning), so I would consider them as being reserved for future core usage.

**Comment by markfairbanks** [comment]
Thanks for the info @alexander-beedie 😄 

> The `<<` and `>>` operators may get used for bitshift ops (their usual meaning), so I would consider them as being reserved for future polars/core use, rather than as a potential API extension point for external packages - otherwise you might get broken again :(

This has been my worry and why I hadn't updated `tidypolars` with this yet. The new feature you implemented in https://github.com/pola-rs/polars/pull/5531 seems like a much better way to go.

**Comment by alexander-beedie** [comment]
> This has been my worry and why I hadn't updated `tidypolars` with this yet. The new feature you implemented in [pola-rs/polars#5531](https://github.com/pola-rs/polars/pull/5531) seems like a much better way to go.

And it's available now - new version released a few hours ago ;)
